### DISCUSSION TITLE
New ideas for future versions of sternalboard

### DISCUSSION THREAD

**Discussion by jonakls** [ShieldCommunity/SternalBoard#48] [Ideas]
I enabled this space so that you can give your ideas for future releases

**Comment by Xxyuri2005xX** [comment]
- Custom placeholders creation to use with SternalBoard (example: %online% -> returns the number of online players) with as parameters in the creation: placeholder (%time%), update(in seconds, 1), return (the part of the code that returns what is of interest)
- Placeholders update method

**Reply by xism4** [reply]
Yes, i can add that but why? All servers basicaly use PlaceHolderAPI and player expansion.

**Reply by Xxyuri2005xX** [reply]
the idea would be useful if a person wants to implement support for SternalBoard in a minigame plugin (an example is a bedwars or skywars plugin), the custom placeholder system would allow you to create for example a placeholder "%skywars_on%" with a refresh interval of 1000ms that returns to you the number of players inside a certain arena (like 4)

**Reply by xism4** [reply]
But this placeholders should be created by the plugin, the scoreboard just parses the placeholders.
